petastorm/tests/conftest.py

```diff
@@ -28,14 +28,15 @@
 
 ROWS_COUNT = 100
 
+_CACHE_FAKE_DATASET_OPTION_SHORT = '-Y'
```

Comment:
Just curious why `-Y` and not something like `-C` for cache?

Reply:
It was -Y for sYnthetic. Small '-c' is taken. Want to try and avoid conflicts with whatever pytest plugins we could see and -Y sounded quite isoteric.